### DISCUSSION TITLE
position of the sun in the sky

### DISCUSSION THREAD

**Discussion by MarcosDionizio** [Stellarium/stellarium#3244] [General]
Hi, I would like to know what function I could use to indicate the position of the Sun (declination and right ascension or height and azimuth) in the sky for a given latitude and longitude.

**Comment by gzotti** [comment]
You click on the sun, set location and time, and read data off screen.

**Reply by MarcosDionizio** [reply]
I would like a mathematical function to use in a study about  the sun's movement. Therefore, I would use this function in some simulations.

**Reply by gzotti** [reply]
Have a look e.g. into the analemma script for some ideas then.
